compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/BLangTestablePackage.java

```diff
@@ -29,6 +29,8 @@ public class BLangTestablePackage extends BLangPackage {
     // Semantic Data
     //Map to maintain all the mock functions
     private Map<String, String> mockFunctionNamesMap = new HashMap<>();
+
+    private final Map<String, String> mockFunctionTypeMap = new HashMap<>();
```

Comment:
Shall we use a Map<String, boolean> instead? it's either Legacy or latest right?

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/42091/commits/ef0fc467445c85fdbe56a9810b1572f5ac988205